.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Usually you'd exclude any configs that are personal preference (such as editors) in either a [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) or the project `.git/info/exclude` file.

That said, since our community favors PyCharm - probably best to keep others from including it on accident. You may also need to add `*.iml` too.